src/webui/layouts/home/Home.tsx

```diff
@@ -1,9 +1,10 @@
 
 import React from 'react';
 
-import './Home.css';
 import { Ruler } from '../../designsystem/module.js';
```

Comment:
The import should not end with `.js`. I left it there probably, but would you remove it for me?